packages/components/Callout/macos/CalloutWindow.swift

```diff
@@ -22,11 +22,32 @@ class CalloutWindow: NSWindow {
 		// Dismiss the Callout if the window is no longer active.
 		NotificationCenter.default.addObserver(self, selector: #selector(dismissCallout), name: NSApplication.didResignActiveNotification, object: nil)
 
-		// Dismiss the Callout if the user touched/clicked outside the bounds of the callout.
+		// Dismiss the Callout if the user clicked outside the Callout Window or any of our child windows.
 		mouseEventMonitor = NSEvent.addLocalMonitorForEvents(matching: .leftMouseUp, handler: { [weak self] (event) -> NSEvent? in
-			if (event.window != self) {
+			guard let clickedWindow = event.window else {
+				return event
+			}
+
+			var shouldDismissCallout = false
```

Comment:
should this be true then you don't need line 43?

Reply:
I could probably also chop off line 36 and 36. I'll sleep on it, my brain is tired of looking at long if/else trees 🤣

Reply:
Actually.. seems to work.